### PR TITLE
sorted signals

### PR DESCRIPTION
**Pull request by Red-Eyed**
Hello, just made PR that shows signals in right order
I've built it and run so it works.

- Made signals sorted.
- Made right correspondence  between signal value and name
- Refactored

image

### CODE REVIEW

File: src/interface.c [hunk 1]
```diff
@@ -1279,22 +1280,73 @@ static void draw_processes(struct device_info *dev_info,
...
 }
 
-static const char *signalsName[] = {
-    "Cancel",  "SIGABRT", "SIGALRM",   "SIGBUS",  "SIGCHLD", "SIGCONT",
-    "SIGFPE",  "SIGHUP",  "SIGILL",    "SIGINT",  "SIGKILL", "SIGPIPE",
-    "SIGQUIT", "SIGSEGV", "SIGSTOP",   "SIGTERM", "SIGTSTP", "SIGTTIN",
-    "SIGTTOU", "SIGUSR1", "SIGUSR2",   "SIGPOLL", "SIGPROF", "SIGSYS",
-    "SIGTRAP", "SIGURG",  "SIGVTALRM", "SIGXCPU", "SIGXFSZ",
+static const char *signalNames[] = {
+  "SIGHUP",
```
Comment (Syllo) [inline src/interface.c]: Keep `"Cancel"` as the fist option please.
The code below assumes that the first of the list is cancel.

File: src/interface.c [hunk 2]
```diff
@@ -1650,7 +1702,7 @@ static void option_do_kill(struct nvtop_interface *inter) {
   if (inter->process.option_window.selected_row == 0)
```
Comment (Syllo) [inline src/interface.c]: This is the cancel-specific bit.

File: src/interface.c (hunk 1 above)
Comment (Red-Eyed) [reply src/interface.c]: Okay, but can I define `"Cancel"` as `SIGKILL` ?

File: src/interface.c (hunk 2 above)
Comment (Red-Eyed) [reply src/interface.c]: Oh, I got it. I thought that `"Cancel"` meant to kill process.

File: src/interface.c (hunk 1 above)
Comment (Red-Eyed) [reply src/interface.c]: Done

**Comment by Red-Eyed** [comment]
So, I've added back "Cancel"

**Comment by Syllo** [comment]
Thank you, merged as part of 1a985741e3b59d1cd2b335ac2c13c3b27f8d6988

**Pull request closed without merging**